troubleshooting/pbs-troubleshooting.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: page_v2
+title: Prebid Server Troubleshooting 
+description: Prebid Server Troubleshooting
+sidebarType: 7
+---
+
+# Prebid Server Troubleshooting
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Getting more debug info
+
+There are several ways to get more debug info from Prebid Server:
+
+### Direct Prebid Server Invocation
+
+One of these parameters needs to be added to the OpenRTB:
+- `"test":1` -- this will inform the bidders that this should be treated as a test (non-billable) request, and also provide additional debug info in the OpenRTB response.
+- `"debug":1` -- this just adds the additional debug info.
```

Comment:
Is the additional info provided from "debug":1 the same additional info as provided with "test":1? Or is this more/different info?

Reply:
agreed that wasn't clear. Hopefully that clarifies it a bit.